geotrek/api/v2/filters.py

```diff
@@ -535,6 +535,14 @@ def get_schema_fields(self, view):
                     title=_("Dates after"),
                     description=_("Filter events happening after or during date, format YYYY-MM-DD")
                 )
+            ), Field(
+                name='bookable',
```

Comment:
This causes an error because fields added with `filterset_fields` are automatically added to Swagger

Please remove it from this method and find a way to set description on the automatically generated one : 

![image](https://user-images.githubusercontent.com/26015573/192574908-a3219502-8323-4174-90ef-747fa00ca07e.png)